CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 All notable changes to this project will be documented in this file.
 
 
+## [v0.4.4](https://github.com/puppetlabs/pdk/tree/v0.4.4) (2017-07-18)
+[Full Changelog](https://github.com/puppetlabs/pdk/compare/v0.4.3...v0.4.4)
+
+**Fixed bugs:**
+
+- Validate fails on existing module [\#158](https://github.com/puppetlabs/pdk/issues/158)
+- Can't create module if new module fails to get login [\#157](https://github.com/puppetlabs/pdk/issues/157)
```

Comment:
Yeah I don't know that I care about somewhat duplicate entries, there may be value in users seeing their exact issue under the list of fixed things since the commit/PR message might be more general.